addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/discovery/ZoneDiscoveryService.java

```diff
@@ -75,7 +75,7 @@ public static ThingUID zoneThing(ThingUID bridgeUid, String zoneName) {
      * The available zones are within the {@link DeviceInformationState}. Will will publish those
      * as things via this discovery service instance.
      *
-     * @param state The device information state
+     * @param state     The device information state
```

Comment:
Something went wrong with your formatter.

Reply:
New installed eclipses never apply the ESH formatter settings correctly :/

Reply:
Yes, I have heard about that: https://github.com/eclipse/smarthome/issues/6424